### PR TITLE
Settings: Rename delete-site-options to site-tools

### PR DESCRIPTION
**Pull request by seear**
Part of #12169.

No visual changes.

Give the `<DeleteSiteOptions/>` component an appropriate name that matches its title and purpose.

**To Test**
* Check that _Settings->General_ works displays as usual for any site
* In particular, the _Site Tools_ section should show at the bottom for wpcom sites

### CODE REVIEW

File: client/my-sites/site-settings/site-tools/index.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+			startOver: translate( 'Delete your content' ),
+			deleteSite: translate( 'Delete your site' )
+		};
+
+		if ( ! this.props.hasLoadedSitePurchasesFromServer ) {
+			return null;
+		}
+
+		if ( ! config.isEnabled( 'upgrades/domain-search' ) ) {
+			changeAddressLinkText = translate( 'Change your site address.' );
```
Comment (a8ci18n) [inline client/my-sites/site-settings/site-tools/index.jsx]: Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Change Site Address' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/modify/settings-delete-options)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/site-tools/index.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
+		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
+		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
+		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
+		const strings = {
+			changeSiteAddress: translate( 'Change your site address' ),
+			themeSetup: translate( 'Theme setup' ),
+			startOver: translate( 'Delete your content' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/site-tools/index.jsx]: Hi! I've found a possible matching string that has already been translated **72** times:
`translate( 'Delete all content.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/site-tools/index.jsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+		const selectedSite = this.props.site;
+		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
+		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
+		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
+		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
+		const strings = {
+			changeSiteAddress: translate( 'Change your site address' ),
+			themeSetup: translate( 'Theme setup' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/site-tools/index.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Theme Setup' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/site-tools/index.jsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+		const selectedSite = this.props.site;
+		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
+		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
+		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
+		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
+		const strings = {
+			changeSiteAddress: translate( 'Change your site address' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/site-tools/index.jsx]: Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Change Site Address' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/modify/settings-delete-options)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/site-tools/index.jsx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+	render() {
+		const { translate } = this.props;
+
+		const selectedSite = this.props.site;
+		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
+		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
+		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
+		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
```
Comment (a8ci18n) [inline client/my-sites/site-settings/site-tools/index.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Make your site look like your theme's demo.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (seear) [inline client/my-sites/site-settings/site-tools/index.jsx]: 👎 file rename

File: client/my-sites/site-settings/site-tools/index.jsx (hunk 4 above)
Comment (seear) [inline client/my-sites/site-settings/site-tools/index.jsx]: 👎 file rename

File: client/my-sites/site-settings/site-tools/index.jsx (hunk 3 above)
Comment (seear) [inline client/my-sites/site-settings/site-tools/index.jsx]: 👎 file rename

File: client/my-sites/site-settings/site-tools/index.jsx (hunk 2 above)
Comment (seear) [inline client/my-sites/site-settings/site-tools/index.jsx]: 👎 file rename

File: client/my-sites/site-settings/site-tools/index.jsx (hunk 1 above)
Comment (seear) [inline client/my-sites/site-settings/site-tools/index.jsx]: 👎 file rename

**Comment by sirreal** [comment]
I see an extra section _Theme steup_ on this branch. I doubt that's related to your changes, but I wanted to mention it.

Branch:

branch

Stage:

stage

**Comment by seear** [comment]
> Just confirm the extra "Theme setup" section is some some feature under a dev flag

Yeah, well spotted. That is feature-flagged by [`config.isEnabled( 'settings/theme-setup' )`](https://github.com/Automattic/wp-calypso/blob/7cdb95992725be0d4e68fc19b4e441dbcb32b0ac/client/my-sites/site-settings/site-tools/index.jsx#L72).

**Pull request merged**